frontend/src/_layouts/base.html

```diff
@@ -113,6 +117,15 @@
       </div>
     </footer>
     <!-- Footer -->
+
```

Comment:
Better put the Backdrop title in the comment here.

Reply:
Added backdrop title in comment